src/Granite/String.hs

```diff
@@ -241,8 +241,8 @@ Each bar is colored differently and labeled with its category name.
 ==== __Example__
 
 @
-let data = [("Apple", 45.2), ("Banana", 38.1), ("Orange", 52.7)]
```

Comment:
Explanation for haddock changes:
I personally find it slightly annoying when I can't copy paste the examples from haddocks to ghci.
since data is keyword in haskell, it can't be used as identifier.
Also quotes, backslashes are treated as special chars in haddock markup, so need to be escaped if you want them to appear in the generated html docs.